airflow/example_dags/example_params_ui_tutorial.py

```diff
@@ -159,9 +156,10 @@
         ),
         # Fields can be required or not. If the defined fields are typed they are getting required by default
         # (else they would not pass JSON schema validation) - to make typed fields optional you must
-        # permit the optional "null" type
+        # permit the optional "null" type.
+        # You can omit a default value if the DAG is triggered manually
         "required_field": Param(
-            "You can not trigger if no text is given here!",
+            # Do not provide a default value but form will enforce a value supplied by users
```

Comment:
This “do not” here reads like “you can not” to me. Is this intended?

Reply:
Thanks. Good point. Adjusted the comment ... and as I had my hand's dirty on this added some more tests for valid and invalid params parsing as well.